artifactory/cli.go

```diff
@@ -1486,10 +1485,16 @@ func shouldSkipGoFlagParsing() bool {
 	}
 
 	if !exists {
-		return false
+		// If missing in the root project, check in the home dir
+		jfrogRootDir, err = config.GetJfrogHomeDir()
```

Comment:
I think that the name of the * jfrogRootDir* variable should be changed, because it is actually not the root dir.
What do you think?
How about *ProjectConfPath*?

Reply:
This one I think should be jfrogHomeDir since it represents the home dir. The jfrogRootDir above this I renamed to projectDir. Let me know what you think.